.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         java:
-          - "8"
+          - "17"
```

Comment:
Let's continue to build with 8 as well.

Reply:
Later in this workflow, we can change the deploy step to only `if: {{ matrix.java == "8" }}`

Reply:
Re-added Java 8 build; build artifacts are now named `SportPaper-Java-8.jar` or `SportPaper-Java-17.jar` but let me know if there's a way you'd prefer that to be formatted. Also added the check to only deploy the Java 8 build since it seems to work fine despite the newer SpecialSource version.